### ISSUE TITLE
Duplicating modules in page layout does not work properly

### ISSUE THREAD

**Issue by stefansl** [contao/core-bundle#520]
If I duplicate a module in the page layout, the column value after saving is wrong.

**How to reproduce:**
In your page layout duplicate a module from the type "Main column", "Footer", but NOT "Header".
After saving your column type changed to "Header".
Appears in Contao 4.2 beta and the release from today.

**The problem**
Incrementing the array key in the name value in /assets/contao/js/core.js does not work for the column field. The getFirst() function works not recursivly and the select element is not found. 

**Solution**
In /assets/contao/js/core.js:1780 change getFirst to getElement:
```javascript
				if (select = childs[j].getElement('select')) {
					select.name = select.name.replace(/\[[0-9]+]/g, '[' + i + ']');
				}
```

**Comment by leofeyer** [comment]
I have just tried it in the latest version (Contao 4.2.0) and I could not reproduce the issue. I have duplicated a module in the left column and saved the form. The new module was still in the left column.

**Comment by stefansl** [comment]
That's strange. I tried a third installation and captured it. You can see it here. Maybe you did something differently?  
See: https://www.youtube.com/watch?v=20n5wxyC70o

**Comment by aschempp** [comment]
I noticed this issue today too but did not yet search for a solution.

**Comment by stefansl** [comment]
The root problem is, that stylect.js wraps the select element and therefore select = childs[j].getElement('select') will match for the column select

**Comment by leofeyer** [comment]
Fixed in contao-components/contao@6953f786105b8d63364b3b27d3c38738d5c1828a.
